City-of-Helsinki/helsinki-paatokset
# PP-0: Drupal core and modules updates for 22.11.2021

This includes a Drupal core security update, module updates and updates to hdbt + helfi_platform_config.

**To test**
- Checkout branch
- Run `make fresh` or `drush cr;drush updb;drush cim -y`
- Check that all modules are up-to-date: https://helsinki-paatokset.docker.so/en/admin/reports/updates
- Check that content from migrations doesn't break (policymakers, etc)
- Check if these blocks are now available after the hdbt / helfi_platform_config module updates: https://helsinkisolutionoffice.atlassian.net/browse/PP-241
